docs/src/content/components/text-paragraph.mdx

```diff
@@ -5,27 +5,93 @@ route: /components/text-paragraph
 
 import { P } from '@pluralsight/ps-design-system-text'
 
-# Paragraphs
+# Paragraph
 
 <TableOfContents {...props} />
 
 ## Examples
 
-Try to use common paragraph style when possible.
+### Sizes
+
+Use the `size` prop to set the size.
+
+```typescript
+import { P } from '@pluralsight/ps-design-system-text'
+import React from 'react'
+
+const Example: React.FC = props => {
+  return (
+    <React.Fragment>
+      <P>
+        Normal Paragraph - Lorem ipsum dolor sit amet adipisicing elit, sed do
+        eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad
+        minim veniam, quis nostrud exercitation nisi ut aliquip ex ea commodo
+        consequat.Lorem ipsum dolor sit amet consectetur adipisicing elit, sed
+        do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+      </P>
+      <P>
+        Normal Paragraph - Lorem ipsum dolor sit amet adipisicing elit, sed do
+        eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad
+        minim veniam, quis nostrud exercitation nisi ut aliquip ex ea commodo
+        consequat.Lorem ipsum dolor sit amet consectetur adipisicing elit, sed
+        do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+      </P>
+
+      <P size={P.sizes.large}>
+        Large Paragraph - Lorem ipsum dolor sit amet adipisicing elit, sed do
+        eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad
+        minim veniam, quis nostrud exercitation nisi ut aliquip ex ea commodo
+        consequat.Lorem ipsum dolor sit amet consectetur adipisicing elit, sed
+        do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+      </P>
+      <P size={P.sizes.large}>
+        Large Paragraph - Lorem ipsum dolor sit amet adipisicing elit, sed do
+        eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad
+        minim veniam, quis nostrud exercitation nisi ut aliquip ex ea commodo
+        consequat.Lorem ipsum dolor sit amet consectetur adipisicing elit, sed
+        do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+      </P>
+
+      <P size={P.sizes.small}>
+        Small Paragraph - Lorem ipsum dolor sit amet adipisicing elit, sed do
+        eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad
+        minim veniam, quis nostrud exercitation nisi ut aliquip ex ea commodo
+        consequat.Lorem ipsum dolor sit amet consectetur adipisicing elit, sed
+        do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+      </P>
+      <P size={P.sizes.small}>
+        Small Paragraph - Lorem ipsum dolor sit amet adipisicing elit, sed do
+        eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad
+        minim veniam, quis nostrud exercitation nisi ut aliquip ex ea commodo
+        consequat.Lorem ipsum dolor sit amet consectetur adipisicing elit, sed
+        do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+      </P>
```

Comment:
```suggestion
      <P size={P.sizes.small}>
        *Small Paragraph* - Lorem ipsum dolor sit amet adipisicing elit, sed do
        eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad
        minim veniam, quis nostrud exercitation nisi ut aliquip ex ea commodo
        consequat.Lorem ipsum dolor sit amet consectetur adipisicing elit, sed
        do eiusmod tempor incididunt ut labore et dolore magna aliqua.
      </P>

      <P>
        *Normal Paragraph* - Lorem ipsum dolor sit amet adipisicing elit, sed do
        eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad
        minim veniam, quis nostrud exercitation nisi ut aliquip ex ea commodo
        consequat.Lorem ipsum dolor sit amet consectetur adipisicing elit, sed
        do eiusmod tempor incididunt ut labore et dolore magna aliqua.
      </P>

      <P size={P.sizes.large}>
        *Large Paragraph* - Lorem ipsum dolor sit amet adipisicing elit, sed do
        eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad
        minim veniam, quis nostrud exercitation nisi ut aliquip ex ea commodo
        consequat.Lorem ipsum dolor sit amet consectetur adipisicing elit, sed
        do eiusmod tempor incididunt ut labore et dolore magna aliqua.
      </P>
```

Long example and difficult to visually parse.  Perhaps this will help?